### PR TITLE
Profile Page - Regen TImer

### PR DESCRIPTION
**Pull request by MiteyDrill**
image
Regen updates every second on profile.

Health/Chakra/Stamina don't update every second to account for the regen timer reaching 0, but I can probably work on that later today or tomorrow.

### CODE REVIEW

**Comment by MiteyDrill** [comment]
Forgot to remove some testing variables will fix later.

**Comment by MiteyDrill** [comment]
Oh you know what, when I go AFK there might be a regen cut. But i'll have to see. I'm not actually sure how regen cut works

File: profile.php [hunk 1]
```diff
@@ -201,21 +199,88 @@ function userProfile() {
...
+		//for some reason every other tick the javascript regen is ahead of the actual regen?
+		//can't figure out why? its like the Regen changes every other minute in intervals or it doubles
+		//can't find the error with my script
+		//can't seem to find out where the error is, need help.
+
+		</script>
+		";
+
+    echo "<label style='width:9.2em;'>Regen Timer:</label>
+		<p id='regentimer' style='display:inline'>" . (60 - $time_since_last_regen) . "</p>
```
Comment (levimeahan) [inline profile.php:275]:
```suggestion
		<span id='regentimer'>" . (60 - $time_since_last_regen) . "</span>

```
If we're going to use `display:inline`, may as well use a span here

File: profile.php [hunk 2]
```diff
@@ -159,37 +159,35 @@ function userProfile() {
...
+
 		$health_percent = round(($player->health / $player->max_health) * 100);
 		$chakra_percent = round(($player->chakra / $player->max_chakra) * 100);
 		$stamina_percent = round(($player->stamina / $player->max_stamina) * 100);
 
 		echo "<td style='width:50%;'>
-		<label style='width:6.7em;'>Health:</label>" . 
-			sprintf("%.2f", $player->health) . '/' . sprintf("%.2f", $player->max_health) . "<br />" .
+		<label style='width:6.7em;'>Health:</label>" .
+			"<p id='health' style='display: inline'>". sprintf("%.2f", $player->health) . '/' . sprintf("%.2f", $player->max_health) . "</p><br />" .
```
Comment (levimeahan) [inline profile.php:178]:
```suggestion
			"<span id='health'>". sprintf("%.2f", $player->health) . '/' . sprintf("%.2f", $player->max_health) . "</span><br />" .

```

File: profile.php [hunk 3]
```diff
@@ -159,37 +159,35 @@ function userProfile() {
...
 			"<div style='height:6px;width:250px;border-style:solid;border-width:1px;'>" .
-			"<div style='background-color:#C00000;height:6px;width:" . $health_percent . "%;' /></div>" . "</div>" .
-		"<label style='width:6.7em;'>Chakra:</label>" . 
-			sprintf("%.2f", $player->chakra) . '/' . sprintf("%.2f", $player->max_chakra) . "<br />" .
+			"<div id='healthbar' style='background-color:#C00000;height:6px;width:" . $health_percent . "%;' /></div>" . "</div>" .
+		"<label style='width:6.7em;'>Chakra:</label>" .
+			"<p id='chakra' style='display: inline'>". sprintf("%.2f", $player->chakra) . '/' . sprintf("%.2f", $player->max_chakra) . "</p><br />" .
```
Comment (levimeahan) [inline profile.php:182]:
```suggestion
			"<span id='chakra'>". sprintf("%.2f", $player->chakra) . '/' . sprintf("%.2f", $player->max_chakra) . "</span><br />" .

```

File: profile.php [hunk 4]
```diff
@@ -159,37 +159,35 @@ function userProfile() {
...
 			"<div style='height:6px;width:250px;border-style:solid;border-width:1px;'>" .
-			"<div style='background-color:#0000B0;height:6px;width:" . $chakra_percent . "%;' /></div>" . "</div>" .
-		"<label style='width:6.7em;'>Stamina:</label>" . 
-			sprintf("%.2f", $player->stamina) . '/' . sprintf("%.2f", $player->max_stamina) . "<br />" .
+			"<div id='chakrabar' style='background-color:#0000B0;height:6px;width:" . $chakra_percent . "%;' /></div>" . "</div>" .
+		"<label style='width:6.7em;'>Stamina:</label>" .
+			"<p id='stamina' style='display: inline'>". sprintf("%.2f", $player->stamina) . '/' . sprintf("%.2f", $player->max_stamina) . "</p><br />" .
```
Comment (levimeahan) [inline profile.php:186]:
```suggestion
			"<span id='stamina'>". sprintf("%.2f", $player->stamina) . '/' . sprintf("%.2f", $player->max_stamina) . "</span><br />" .

```

**Review by levimeahan — APPROVED** [review]

**Pull request merged**